tls/extensions/s2n_cert_authorities.c

```diff
@@ -95,6 +114,18 @@ int s2n_cert_authorities_send(struct s2n_connection *conn, struct s2n_stuffer *o
     return S2N_SUCCESS;
 }
 
+int s2n_cert_authorities_recv(struct s2n_connection *conn, struct s2n_stuffer *in)
+{
+    POSIX_ENSURE_REF(conn);
+    POSIX_ENSURE_REF(conn->config);
+    POSIX_ENSURE_EQ(conn->mode, S2N_CLIENT);
```

Comment:
Will make this a no-op if it's not on the client... s2n never sends *to* the server today, but other implementations could. Not sure we'd get called for ClientHello containing the cert authorities but better to be on the safe side.